src/Microsoft.Kiota.Abstractions.csproj

```diff
@@ -6,7 +6,7 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <AssemblyTitle>Kiota Abstractions Library for dotnet</AssemblyTitle>
     <Authors>Microsoft</Authors>
-    <TargetFrameworks>netstandard2.0;netstandard2.1;</TargetFrameworks>
```

Comment:
@andrueastman I have attempted to keep the original value here, but I'm unable to produce a `csproj` in `std-uritemplate` which keeps the compilation happy 😞 .

I need to ask for help to get this right ...

Reply:
Any chance you can confirm the specific error you get when you do this? When I add back the `netstandard2.1;` target, the code compiles without issue locally.

Reply:
Thanks a lot for double-checking @andrueastman !
You are right the problem was local 🤦  I didn't have the `netstandard2.1` installed 😞 now everything works as expected.